src/runtime/recipe/handle.ts

```diff
@@ -209,6 +212,8 @@ export class Handle implements Comparable<Handle> {
   set mappedType(mappedType: Type) { this._mappedType = mappedType; }
   get immediateValue() { return this._immediateValue; }
   set immediateValue(value: ParticleSpec) { this._immediateValue = value; }
+  get ttl() { return this._ttl; }
```

Comment:
If the getters and setters make this `_ttl` field effectively public, can we avoid implementing them and just make it public?

Reply:
I did it this way, because it's more consistent with the rest of the file...

Reply:
Interesting. I think we (as a team) should make a decision about getters and setters. We seem to use them a lot but I'm not sure why (I understand why we might have a getter but not a setter, or even a setter but not a getter, but if we have both the value is effectively public).